### ISSUE TITLE
start-gateway-config.sh  not using DIAGNOSTICS_JSON_URL

### ISSUE THREAD

**Issue by softlion** [NebraLtd/hm-config#263]
`start-gateway-config.sh`  is using the hardcoded string `http://diagnostics/json` instead of `DIAGNOSTICS_JSON_URL`

**Comment by shawaj** [comment]
Iirc diagnostics json url is only assigned in the python part of the library so don't thing is available at this stage of startup. 

I might be wrong however, as it's been a while since I've dug into this area

**Comment by shawaj** [comment]
Yeah it's set here but only for the python library...https://github.com/NebraLtd/hm-config/blob/a198b83776998a75044a0dfd88086971017575e2/gatewayconfig/__main__.py#L21
